### PR TITLE
Fix timecode scan

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -350,7 +350,7 @@ _setup_vrecord_process(){
         fi
     }
     # set up input and playback
-    unset GRAB_INPUT
+    GRAB_INPUT=()
     GRAB_INPUT+=("${EXTRAINPUTOPTIONS[@]}")
```
Comment (retokromer) [inline vrecord:354]: Would `GRAB_INPUT=("${EXTRAINPUTOPTIONS[@]}")` nor be the same, but shorter for both lines?

Comment (dericed) [reply vrecord:354]: That would be the same and shorter.

**Review by privatezero — APPROVED** [review]
This all seemed to work on my end, although I didn't have a test tape close at hand that has time code

**Review by retokromer — APPROVED** [review]
Works on my end (tested on Debian 12.1, Ubuntu 23.04 and 22.04.3 LTS and on macOS 13.4.1).

**Pull request merged**